README.md

```diff
@@ -225,11 +240,11 @@ and temporal autocorrelation:
 
 ``` r
 part_GLS$overall$t.test
-#>                          Est          SE     t.stat       pval.t
-#> (Intercept)       0.30873438 0.007175747 43.0247021 0.000000e+00
-#> coords.x          0.10018682 0.012176186  8.2280948 2.139898e-16
-#> coords.y          0.19825454 0.012191562 16.2616200 1.048738e-58
-#> coords.x:coords.y 0.02048618 0.020733363  0.9880781 3.231384e-01
+#>                          Est          SE    t.stat       pval.t
+#> (Intercept)       0.30903199 0.007183905 43.017274 0.000000e+00
+#> coords.x          0.10325420 0.012208016  8.457902 3.104167e-17
+#> coords.y          0.19481642 0.012191460 15.979745 8.939526e-57
+#> coords.x:coords.y 0.03040168 0.020736468  1.466097 1.426533e-01
```

Comment:
These changes (and below) are due to the randomness of the README example code. I should probably set a seed to prevent this changing with each new render.